pkg/migrate/migrate.go

```diff
@@ -124,46 +216,50 @@ func (m migrator) migrate(ctx context.Context) error {
 			newKey := newPrefix + oldKey[len(oldPrefix):]
 			beforeKV[newKey] = v.Value
```

Comment:
beforeKV should be delete if there is no need to check data before and after migrate.